docs/reference/migration/migrate_7_0/indices.asciidoc

```diff
@@ -78,3 +78,12 @@ The parent circuit breaker defines a new setting `indices.breaker.total.use_real
 heap memory instead of only considering the reserved memory by child circuit breakers. When this
 setting is `true`, the default parent breaker limit also changes from 70% to 95% of the JVM heap size.
 The previous behavior can be restored by setting `indices.breaker.total.use_real_memory` to `false`.
+
+==== `fix` value for `index.shard.check_on_startup` is removed
+
+`elasticsearch-shard remove-corrupted-segments` tool has to be used instead of
+`index.shard.check_on_startup: fix` setting.
+
+==== `elasticsearch-translog` tool merged into `elasticsearch-shard`
+
+Instead of `elasticsearch-translog` tool you should use `elasticsearch-shard truncate-translog`.
```

Comment:
Similarly:

Use the `elasticsearch-shard` tool to remove corrupted translog data.